umade-server/src/main/java/com/umade/utils/database/UsersDAO.java

```diff
@@ -36,4 +36,24 @@ public void add(String name, String email, String password, String phone, String
                 " (name, email, password, phone, gender, defectType, address, role) VALUES (?, ?, ?, ?, ?, ?, ?, 'guest')",     // When a new user sign up, role will be 'guest' by default
                 name, email, password, phone, gender, defectType, address);
     }
+        public boolean passwordChecker(String email, String newPassword){
+        ResultSet rs = executeQueryPreparedStatement("SELECT * FROM " + table + " WHERE email = ?", email);
+        try{
+            if(rs.next()){
+                String currentPassword = rs.getString("password");
+                if(newPassword.equals(currentPassword)){
+                    return true;
+                } else {
+                    return false;
+                }
```

Comment:
Redundant if-else statement
```suggestion
                return newPassword.equals(currentPassword);
```